### PR TITLE
Remove cirq-ft package dependency 

### PR DESCRIPTION
**Pull request by tanujkhattar**
Final PR in Qualtran repo as part of Part of [Qualtran & Cirq-FT Integration](https://docs.google.com/document/d/1miypjcCNt6JV_qAypUVPy5eH3EmKadLqeZomA5QoWK0/edit?usp=sharing&resourcekey=0-ciQ8jPfQmK8gtey2lNKesw)

Once this is merged, Qualtran will have no dependence on the Cirq-FT package and we can go ahead and start deprecations in Cirq-FT.

### CODE REVIEW

**Comment by mpharrigan** [comment]
I think we were relying on some transitive dependencies. `attrs` for one; It looks like the docs (and test?) environment needs `nbconvert`

**Comment by mpharrigan** [comment]
here's the cirq-ft requirements

```
attrs
cachetools>=5.3
ipywidgets
nbconvert
nbformat
```

File: dev_tools/requirements/deps/runtime.txt [hunk 1]
```diff
@@ -1,5 +1,11 @@
-cirq-core==1.3.0.dev20231010175755
-cirq-ft==1.3.0.dev20231010175755
+cirq-core==1.3.0.dev20231018023458
+attrs
+cachetools>=5.3
+
+# docs
+ipywidgets
+nbconvert
+nbformat
```
Comment (mpharrigan) [inline dev_tools/requirements/deps/runtime.txt:8]: caaan we put these in `deps/docs.txt`

Comment (tanujkhattar) [reply dev_tools/requirements/deps/runtime.txt:8]: done

**Comment by tanujkhattar** [comment]
@mpharrigan This is ready for a re-review

**Review by fdmalone — APPROVED** [review]

**Pull request merged**